lambdas/validate-oauth-callback/src/main/java/uk/gov/di/ipv/core/validateoauthcallback/ValidateOAuthCallbackHandler.java

```diff
@@ -112,6 +113,10 @@ public Map<String, Object> handleRequest(CredentialIssuerRequestDto request, Con
 
             ipvSessionService.updateIpvSession(ipvSessionItem);
 
+            var mapMessage =
+                    new StringMapMessage().with("message", "Successfully validated oauth callback");
```

Comment:
Might be useful to add the criId of the request into this log message?

Reply:
Good idea I've added that now